tests/tests.js

```diff
@@ -76,7 +76,6 @@ QUnit.test('iframeFullyQualifiedName: multiple layers', function(assert) {
   childFrameBody.appendChild(grandChildFrame1);
   childFrameBody.appendChild(grandChildFrame2);
   assert.equal(serializer.iframeFullyQualifiedName(grandChildFrame1.contentWindow), '0.0.0');
-  assert.equal(serializer.iframeFullyQualifiedName(grandChildFrame1.contentWindow), '0.0.0');
```

Comment:
I think I accidentally copied the same line twice, so I deleted the second one.  This is another instance of me lumping a small change into a pull request (as mentioned above).